packages/clerk-js/src/core/clerk.ts

```diff
@@ -949,7 +953,36 @@ export default class Clerk implements ClerkInterface {
     this.#componentControls?.updateProps(props);
   };
 
-  #loadInStandardBrowser = async (): Promise<void> => {
+  #syncWithPrimary = () => {
+    const q = new URLSearchParams({
+      redirect_url: `${window.location.href}?synced=true`,
+    });
+    window.location.replace(new URL(`/v1/client/sync?${q.toString()}`, `https://${this.domain}`).toString());
+  };
+
+  #handleSyncedQueryParam = () => {
+    function stripSyncedQueryParam(q: URLSearchParams) {
+      q.delete('synced');
```

Comment:
:upside_down_face: Wonder if we should name this parameter `__clerk_synced` to avoid any naming collisions.